src/components/ListItem.jsx

```diff
@@ -38,6 +38,14 @@ export function ListItem({ name, itemId, dateLastPurchased }) {
 		});
 	};
 
+	// When remove button is clicked, a confirm window pops up, and when user confirms, deleteItem() is called
+	const handleDelete = async (e) => {
+		const listId = localStorage.getItem('tcl-shopping-list-token');
```

Comment:
Not part of the criteria, but `const listId` is used by both `handleDelete` and `handleCheck` so I think it would be good to make `listId` a global variable.